lib/cherrypy/wsgiserver/wsgiserver2.py

```diff
@@ -78,24 +96,6 @@ def my_crazy_app(environ, start_response):
            'WSGIPathInfoDispatcher', 'get_ssl_adapter_class',
            'socket_errors_to_ignore']
 
-import os
-try:
-    import queue
-except:
-    import Queue as queue
-import re
-import email.utils
-import socket
-import sys
-import threading
-import time
-import traceback as traceback_
-import operator
-from urllib import unquote
-from urlparse import urlparse
-import warnings
-import errno
-import logging
```

Comment:
Moving these imports up, and some added blank lines below, strikes me as unnecessary and - if we were planning to keep using Cherrypy all this would make it very difficult to upgrade to new Cherrypy versions.  However, since we're not (keeping it) ... I guess it doesn't really matter!

Reply:
all part of the pep8 rollercoaster!